src/phoenix/experimental/evals/functions/classify.py

```diff
@@ -628,3 +631,79 @@ def _default_openai_function(
             "required": required,
         },
     }
+
+
+class RunEvalsPayload(TypedDict):
```

Comment:
Unless there's a specific advantage, I would recommend sticking to its duller sibling `NamedTuple`. But that's just a suggestion.

> Zen of Python:
> There should be one-- and preferably only one --obvious way to do it.

Reply:
addressed in 9deba831